lib/sentry/transaction.ex

```diff
@@ -0,0 +1,75 @@
+defmodule Sentry.Transaction do
+  @type t() :: %__MODULE__{}
+
+  alias Sentry.{Config, UUID}
+
+  @enforce_keys ~w(event_id span_id spans)a
```

Comment:
Same number of characters and easier for readers less familiar with Elixir. Not a long list, so I don't think `~w` is worth it 🙃 

```suggestion
  @enforce_keys [:event_id, :span_id, :spans]
```